guides/earn-grc.htm

```diff
@@ -153,6 +153,11 @@ <h5 class="text-center">Q: My Wallet Is Not Staking</h5>
                 <h5 class="text-center mt-5">Q: My Balance is Too Low To Send a Beacon</h5>
                     <li>You need around 1 GRC to send a beacon. If you recently received some GRC, you will need to wait a few minutes for the transaction to be confirmed before using that GRC for a beacon.</li>
                     <li>You can use faucets to get free GRC for sending a Beacon: <a href="https://www.gridcoinstats.eu/faucet">Gridcoinstats.eu</a>, <a href="https://gridcoin.ch/faucet">Gridcoin.ch</a>. Our discord is a great place to ask for a few free GRC to get started</li>
+                <h5 class="text-center mt-5">Q: I See Multiple CPIDs or Receive a CPID Mismatch Error</h5>

```

Comment:
Probably worth also mentioning here that using an account manager can help with this

Reply:
A user in this situation who is looking for a solution is just going to get _more_ confused if we tell them "After you've setup everything for solo crunching, now go setup an account manager and connect your BOINC client to it in a different way than the rest of the guide told you to connect to projects. I love BAM and think it's an easy way to use BOINC, but it does have some bugs and using it isn't documented or the suggested path earlier in the guides. There should be one, single default path which should be as simple as possible. All the user has to do this this instance, generally, is wait.

Reply:
It looks like this will also soon be a thing of the past regardless https://tinyurl.com/boinccpidfix

Reply:
Also I don't think account managers actually do help with split CPID problems. They help with users who use more than one e-mail, but that particular step is made abundantly clear in the guide. You're can get split CPID or changing CPID (due to project removal) whether using BAM or connecting to projects directly.

Reply:
Ah if the BOINC client is going to fix most of this issue soon then yeah we can keep it pretty short here